examples/lighting-app/esp32/main/Button.cpp

```diff
@@ -1,6 +1,7 @@
 /*
  *
  *    Copyright (c) 2022 Project CHIP Authors
```

Comment:
if we're updating this, should we just make this 2023 too?
```suggestion
 *    Copyright (c) 2022-2023 Project CHIP Authors
```